raiden-dapp/src/views/DisclaimerRoute.vue

```diff
@@ -1,10 +1,52 @@
 <template>
   <div data-cy="disclaimer" class="disclaimer">
     <div class="disclaimer__content">
-      <div class="disclaimer__content__paragraphs font-weight-light">
-        <p v-for="(paragraph, index) in $t('disclaimer.paragraphs')" :key="index">
-          {{ paragraph }}
-        </p>
+      <div class="disclaimer__content__text">
+        <span class="disclaimer__content__text__header">
+          {{ $t('disclaimer.user-info.header') }}
+        </span>
+        <span class="disclaimer__content__text__body">{{ $t('disclaimer.user-info.body') }}</span>
+        <div>
+          <ul
+            v-for="(bulletPoint, index) in $t('disclaimer.user-info.bullet-points')"
+            :key="index"
+            class="disclaimer__content__text__bullet-points"
+          >
+            <li>
+              {{ bulletPoint.text }}
+              <a :href="disclaimerUrlMappings[bulletPoint.url]" target="_blank">
+                {{ bulletPoint.link }}
+              </a>
+            </li>
+          </ul>
+        </div>
+        <span v-if="imprint" class="disclaimer__content__text__header">
+          {{ $t('disclaimer.terms.header') }}
```

Comment:
`if imprint then print terms` - this is a little confusing. Shouldn't what is called `terms` now be actually called `imprint`? Or does the `imprint` consists of the `terms`? I'm not to sure about the last, but if not or if it only consists of the `terms` and nothing more, I would consider to rename them to the same.

Reply:
You are right, Christian informed me that there are actually two different texts now, these will need to be adjusted in the footer as well.